Mathlib/Order/KrullDimension.lean

```diff
@@ -395,22 +395,55 @@ lemma krullDim_eq_of_orderIso (f : α ≃o β) : krullDim α = krullDim β :=
 
 /--
 The Krull dimension is the supremum of the elements' heights.
+
+See `krullDim_eq_iSup_height` for the more general, but less convenient version without the
+`Nonempty` requirement.
+-/
+lemma krullDim_eq_iSup_height_of_nonempty [Nonempty α] : krullDim α = ⨆ (a : α), height a := by
```

Comment:
Ah wait, this is parsed as
```suggestion
lemma krullDim_eq_iSup_height_of_nonempty [Nonempty α] : krullDim α = ↑(⨆ a : α, height a) := by
```
Probably worth writing it like that, because one would expect the arrow to be pushed inside (and it might become so in the future)